x-pack/plugins/security_solution/public/timelines/store/middlewares/helpers.ts

```diff
@@ -62,3 +63,17 @@ export async function ensureTimelineIsSaved({
   // Make sure we're returning the most updated version of the timeline
   return selectTimelineById(store.getState(), localTimelineId);
 }
+
+export function isHttpFetchError(
```

Comment:
nit: consider moving to it's own file in `x-pack/plugins/security_solution/public/common/utils`